New package: LightMatter v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: LightMatter
- Repository: https://github.com/maurergroup/LightMatter.jl
- Created by: @Snowd1n
- Version: v0.1.0
- Commit: 83f81ccd4e1a511dbbf6320b36bd3b70586b8059
- Reviewed by: @Snowd1n
- Reference: https://github.com/maurergroup/LightMatter.jl/commit/83f81ccd4e1a511dbbf6320b36bd3b70586b8059#commitcomment-162583914
- Description: A 'package' to simulate the electronic and temperature effects of light illumination on a metal.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62d1a40d9863be186e05a180276ed2c5bce58987b4bfa5084b0934bd3731bf358d2314e921007fc77e22db931bfd806e3251b35774d13ef1f6459320d2cd78c2c35da397cbf7fb33d87626469c707abc40412ec40eb5d373cb7afe37ca322944cb8b7292e6de749196ff9cde988c47df1d4fa63652f8d2d1e21c9a61be57e6ca44c0dcfa1be5930c3f31f4369646b308fa9d23770ab9637a6a8fb82a120eeed23cb40682be3956cb2558de0e1c40eb7b4 -->

Snowd1n commented:
I have started building a documenter set-up and now just need to build the CI to host.  I have also started fleshing out the tests as well and added a README. This will all get much more work on the coming week. I just would like it merging before Friday of JuliaCon if possible

Snowd1n commented:
For some reason the package fails to precompile according to the tests but it precompiles fine in a fresh install of julia on my pc, not sure why the test is failing

Snowd1n commented:
This is my fault sorry, on my machine for some reason the file is capitalized but not on the actually git. I have no idea what I have done to do that 🤷‍♂️. I have manually updated on the GitHub page now. Sorry for not catching that

JuliaRegistrator committed:
New package: LightMatter v0.1.0

UUID: b578e399-e75b-4901-9c37-d6aa618b88c4
Repo: https://github.com/maurergroup/LightMatter.jl.git
Tree: 3947c28efc5e69f7c3f4ff483cb4006cfe1df3a9

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
